src/dataflow-types/src/client.rs

```diff
@@ -1111,6 +1134,10 @@ pub mod partitioned {
                         feedback,
                     ))))
                 }
+                // TODO(guswynn): is this the correct implementation?
```

Comment:
It's hard to know without the implementation. If each worker collects the max offsets for the parts that it manages, then we want to integrate the results before returning them.

Reply:
I think that each _storage_ worker will just produce the same results?